pkg/freeze/containerd/pause.go

```diff
@@ -89,27 +92,31 @@ func lookupContainerID(ctx context.Context, conn *grpc.ClientConn, podUID, conta
 		},
 	})
 	if err != nil {
-		return "", err
+		return []string{}, err
 	}
 
 	if len(pods.Items) == 0 {
-		return "", fmt.Errorf("pod %s not found", podUID)
+		return []string{}, fmt.Errorf("pod %s not found", podUID)
 	}
 	pod := pods.Items[0]
 
 	ctrs, err := client.ListContainers(ctx, &cri.ListContainersRequest{Filter: &cri.ContainerFilter{
 		PodSandboxId: pod.Id,
-		LabelSelector: map[string]string{
-			"io.kubernetes.container.name": containerName,
-		},
 	}})
 	if err != nil {
-		return "", err
+		return []string{}, err
+	}
+
+	var ids []string
```

Comment:
we know the size of this array is one less than the size of the containers array, so we might as well pre-allocate (totally premature optimisation, but consistent with rest of the project that tends to do this where possible).

```suggestion
	ids := make([]string, 0, len(ctrs.Containers)-1)
```